runtime/CSharp/runtime/CSharp/Antlr4.Runtime/AntlrInputStream.cs

```diff
@@ -48,11 +48,7 @@ public virtual void Consume()
                 System.Diagnostics.Debug.Assert(LA(1) == IntStreamConstants.EOF);
                 throw new InvalidOperationException("cannot consume EOF");
             }
-            //System.out.println("prev p="+p+", c="+(char)data[p]);
-            if (p < n)
-            {
-                p++;
-            }
+            p++;
```

Comment:
Can you make it an else ?
It would keep the logic readable at no execution cost

Reply:
Sure, I have updated my commit.